test/automl/test_classification.py

```diff
@@ -391,6 +391,22 @@ def test_sparse_matrix_lr(self):
         print(automl_experiment.best_iteration)
         print(automl_experiment.best_estimator)
 
+    def test_metric_for_logging_None(self):
+        # a test case when search_state.metric_for_logging is None
+        import mlflow
```

Comment:
This function won't work here because mlflow is not installed.
You can move the test under test_notebooek_example.py:test_mlflow() where mlflow is installed.